README.rst

```diff
@@ -1,148 +1,186 @@
 
-- Improving signal stability checking each fact with the previous one.
-  [Guillermo Jimenez Prieto]
+API Overview
+============
 
-- Fixing a multitenacy bug. [Guillermo Jimenez Prieto]
+Servers will update their context. The context information contains the description of the CPU, Memory, Disk and
+Network usages.
+
+An example of this operation could be:
+
+::
+
+        curl --include \
+             --request POST \
+             --header "Content-Type: application/json" \
+             --data-binary "{"contextResponses": [
+                {
+                    "contextElement": {
+                        "attributes": [
+                            {
+                                "value": "6",
+                                "name": "users",
+                                "type": "string"
+                            },
+                            {
+                                "value": "1",
+                                "name": "usedMemPct",
+                                "type": "string"
+                            },
+                            {
+                                "value": "0.14",
+                                "name": "cpuLoadPct",
+                                "type": "string"
+                            },
+                            {
+                                "value": "0.856240",
+                                "name": "freeSpacePct",
+                                "type": "string"
+                            }
+                        ],
+                        "id": "Trento:193.205.211.69",
+                        "isPattern": "false",
+                        "type": "host"
+                    },
+                    "statusCode": {
+                        "code": "200",
+                        "reasonPhrase": "OK"
+                    }
+                }
+            ]
+        }" \
+        'http://policymanager-host.org:5000/v1.0/d3fdddc6324c439780a6fd963a9fa148/servers/52415800-8b69-11e0-9b19-734f6af67565'
+
+This message follows the NGSI-10 information model but using JSON format.
+
+
+The response has no body and should return 200 OK.
+
+API Reference Documentation
+---------------------------
+
+- `FIWARE Policy Manager v1 (Apiary)`__
+
+__ `FIWARE Policy Manager - Apiary`_
+
+
+Testing
+=======
 
-- Fixing a multitenacy bug. [Guillermo Jimenez Prieto]
+Unit tests
+----------
 
-- Repaired multitenacy bug with serverid in lists. [Guillermo Jimenez
-  Prieto]
+To execute the unit tests you must have a redis-server and a rabbitmq-server up and running.
 
-v1.1.0 (2015-03-04)
--------------------
+After that, you can execute this folloing commands:
 
-New
-~~~
+::
 
-- New: dev: preparing release @release. [Guillermo Jimenez Prieto]
+    $ python server.py &
+    $ export PYTHONPATH=$PWD
+    $ nosetests -s -v --cover-package=facts --with-cover
 
-v1.0.0 (2015-02-24)
--------------------
+End-to-end tests
+----------------
 
```

Comment:
Apart from the link to admin guide, you should explain how to run a sanity test to check if the service is UP&RUNNING.
*End-to-end tests* is not a good name for this section but this is the name required by the guidelines. :(